contents/docs/how-posthog-works/index.mdx

```diff
@@ -5,6 +5,72 @@ title: Architecture
 The following pages in this section cover [Data model](/how-posthog-works/data-model),  [Ingestion pipeline](/how-posthog-works/ingestion-pipeline), [ClickHouse](/how-posthog-works/clickhouse) and [Querying data](/how-posthog-works/queries). 
 This document gives an overview of the general architecture. How our cloud or a deployed PostHog [Helm chart](https://github.com/PostHog/charts-clickhouse/) works on Kubernetes.
 
+## Zooming right out
+
+There are only a few systems to consider. A Website and API is presented for users. An API for Client Apps. A plugin service for processing events on ingestion. And a worker service for processing events in response to triggers (e.g. timers)
+
+```mermaid
+graph LR
+    u[User]
+    sdk[Client Apps/SDKs]
+    ex[Export Sink]
+    dj[Web/API]
+    p[plugin/worker service]
```

Comment:
I would rephrase this as `ingestion` probably as we actually don't push stuff to plugin/worker service directly